lib/main.cpp

```diff
@@ -0,0 +1,22 @@
+#include "./reader.h"
+
+namespace config_reader {
+    using namespace std;
+
+    std::string *read_config(std::string config_file_path) {
+        int count = 0;
+        std::ifstream inputFile;
+        inputFile.open(config_file_path);
+
+        string line_count[10];
+
```

Comment:
A variable might be better to calculate the number of lines. as the number of lines might change in a config file.